pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -669,6 +669,12 @@ type MirrorHealthCheckSpec struct {
 	// +optional
 	// +nullable
 	Mirror HealthCheckSpec `json:"mirror,omitempty"`
+	// +optional
+	LivenessProbe *ProbeSpec `json:"livenessProbe,omitempty"`
+	// +optional
+	ReadinessProbe *ProbeSpec `json:"readinessProbe,omitempty"`
```

Comment:
Since the readiness probe isn't implemented currently, IMO it's best if we remove this from the API here, and we can add it later if we need it.

Reply:
Ig we should add this to mds, readiness probe is also important for traffic control
and other services are already using it

Reply:
Right. Given that it currently isn't implemented and this PR is just about moving a feature from CephCluster to CephFilesystem, we should leave it out for now and create an issue to add it for follow-up.